thorlcr/graph/thgraphslave.cpp

```diff
@@ -1270,6 +1270,9 @@ bool CSlaveGraph::serializeStats(MemoryBuffer &mb)
     stats.setStatistic(StSizePeakMemory, processActiveInfo.getPeakResidentMemory());
     jobS->querySharedAllocator()->queryRowManager()->reportSummaryStatistics(stats);
 
+    IGraphTempHandler *tempHandler = owner ? queryTempHandler(false) : queryJob().queryTempHandler();
+    if (tempHandler)
+        stats.mergeStatistic(StSizeGraphSpill, tempHandler->getActiveUsageSize());
```

Comment:
What exactly does this value represent?  Is it the size of all spills, both results and internal within a graph, or just one of those?  How does it differ logically from StSizeSpillFile?

Reply:
It's the [peak] size of inter-subgraph spilling, i.e. for a given sub-graph, the inter-subgraph spill size may go up or down as spills are being produced/consumed. At the subgraph level this stat. captures the peak. At the rollup/aggregate (separate PR) level to graph->workflow->job, it will capture the peak of peaks. Such that you can see at a high-level what is the max that was used/required and drill down to find the area of largest use.

It does not capture the intra-activity spilling sizes (e.g. from sort, etc.).
That will be added in a separate JIRA/PR.